test/k8sT/Policies.go

```diff
@@ -857,6 +861,7 @@ var _ = Describe("K8sValidatedPolicyTestAcrossNamespaces", func() {
 
 			By("Running tests WITH Policy / Proxy loaded")
 			testConnectivity(frontendPod.String(), backendSvcIP)
+			policyDeleteAndCheck(ciliumPods, policyPath, policyCmd)
```

Comment:
What happens if something fails in testConnectivity, the policy will never be deleted no? That function should be called on AfterEach or with a defer, if not the cleanup will never happen if the test failed.

Reply:
Good call, fixed. Done in `AfterEach`.